packages/ui-lib/src/theme/variables/Colors.ts

```diff
@@ -549,7 +549,26 @@ export const colorVariables = css`
     --input-color-unit: var(--grey-10);
 
     --input-focused-shadow-color: var(--black-a5);
+    
+    --input-selection-control-default: var(--grey-8);
+    --input-selection-control-default-disabled: var(--grey-6);
+    --input-selection-control-selected: var(--primary-9);
+    --input-selection-control-selected-disabled: var(--primary-8);
+
+  }
 
+  /* Filters */
+  .light-theme, .dark-theme {
+    --filter-button-text-color: var(--grey-11);
+    --filter-button-icon-color: var(--grey-10);
+    --filter-button-background-color: var(--grey-2);
+    --filter-button-stroke-color: var(--grey-a5);
+    --filter-button-shadow-color: var(--primary-a3);
+  }
+
```

Comment:
Any reason to separate these from other component aliases?

Reply:
A couple of reasons that are more future than present.

While the idea is that those color palletes can be switched, there is an occassional need to set different colors depending on the theme. For this reason, as the list of aliases for components gets long, break things up more like this allows us to better group things so you don't have to scroll 100s of lines to find those changes.

Also related to this, I think as the file gets long we may wish to consider splitting off to their own components variable file. So this is a just a way to organise it in the mean time with the intention that it might get moved later.